spec/requests/search_results_controller_spec.rb

```diff
@@ -140,4 +140,44 @@
       end
     end
   end
+
+  describe 'GET #show' do
+    context 'given an existing search result ID' do
+      it 'returns a successful response with JSON' do
+        user = Fabricate(:user)
+        Fabricate.times(20, :search_result, user_id: user.id)
+        authorization_header = create_authorization_header(user: user)
+        search_result = Fabricate(:search_result, user_id: user.id)
+
+        get "/api/v1/search_results/#{search_result.id}", headers: authorization_header
```

Comment:
Optional: 

```suggestion
        get "/api/v1/search_results/1", headers: authorization_header
```

Reply:
@Tuubz Because the search_result generated by the Fabricator will set the id randomly, that's why I reference by `search_result.id` 😄 I will update to stick the id always 1.